lumeer-core/src/main/java/io/lumeer/core/facade/ViewFacade.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *  
+ * Copyright (C) since 2016 the original author or authors.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.core.facade;
+
+import io.lumeer.api.model.Permission;
+import io.lumeer.api.model.Permissions;
+import io.lumeer.api.model.Role;
+import io.lumeer.api.model.View;
+import io.lumeer.core.AuthenticatedUser;
+import io.lumeer.core.PermissionsChecker;
+import io.lumeer.core.WorkspaceKeeper;
+import io.lumeer.core.model.SimplePermission;
+import io.lumeer.storage.api.dao.ViewDao;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+@RequestScoped
```

Comment:
Because we need to initialize `ViewDao` which depends on the project specified in the request. Otherwise, it would be `ApplicationScoped`.

In the future, it would be better to introduce `ProjectScoped` and `OrganizationScoped` in order not to have too many instances of some beans which are not really needed.